geonode/thumbs/background.py

```diff
@@ -182,6 +182,7 @@ def __init__(
         # Slippy Maps specific attributes (to be overwritten in specific background classes)
         self.url = options.get("url", None)
         self.tile_size = options.get("tile_size", 256)
+        self.tms = True if options.get("tms") and options.get("tms") == True else False
```

Comment:
Easier and more readable writing this one as:

`self.tms = options.get("tms", False)`

Reply:
yes, sure @afabiani. I'm going to fix it. However I would check if the value is a boolean.

Reply:
@afabiani it should be like this:

`self.tms = options.get("tms") if options.get("tms") and type(options.get("tms")) is bool else False`